basics/nemo_makeleadfield.m

```diff
@@ -11,39 +11,37 @@
             cfg.feedback              = false;
             cfg.method                = 'singleshell';
             cfg.tissue                = 'brain';
-            headmodel                       = ft_prepare_headmodel(cfg,cfgnemo.bnd(end));
-            
-            
+            headmodel                 = ft_prepare_headmodel(cfg,cfgnemo.bnd(end));
+
+
         case {'dipoli','bemcp','openmeeg'}
             cfg                       = [];
-            cfg.grad                  = cfgnemo.grad_mri;
             cfg.feedback              = false;
             cfg.method                = cfgnemo.headmodelstrategy;
-            
-            headmodel.bnd = cfgnemo.bnd;
+
             switch(cfgnemo.numlayers)
                 case 4
-                    headmodel.cond = [0.33 0.0041 1.79 0.33]; % SI units, all 4 layers
-                    headmodel.cond = [0.33 0.022 1.79 0.33]; % SI units, all 4 layers % <- from Oostendorp
+                    cfg.conductivity = [0.33 0.0041 1.79 0.33]; % SI units, all 4 layers
+                    cfg.conductivity = [0.33 0.022 1.79 0.33]; % SI units, all 4 layers % <- from Oostendorp
                 case 3
-                    headmodel.cond = [0.33 0.0041 0.33]; % SI units, ignore CSF
-                    % headmodel.cond = [0.33 0.022 0.33]; % SI units, ignore CSF % <- from Oostendorp
+                    cfg.conductivity = [0.33 0.0041 0.33]; % SI units, ignore CSF
+                    % headmodel.conductivity = [0.33 0.022 0.33]; % SI units, ignore CSF % <- from Oostendorp
             end
-            
-            headmodel                       = ft_prepare_headmodel(cfg,cfgnemo.bnd);
-            
-            
+
+            headmodel                = ft_prepare_headmodel(cfg, cfgnemo.bnd);
+
+
         case 'openmeeg-old'
             cfg                       = [];
             subjId = [cfgnemo.participant '_' cfgnemo.segmethod '_' num2str(cfgnemo.numlayers) 'layer'];
             headmodel.bnd = cfgnemo.bnd;
             switch(cfgnemo.numlayers)
                 case 4
-                    headmodel.cond = [0.33 0.0041 1.79 0.33]; % SI units, all 4 layers
-                    headmodel.cond = [0.33 0.022 1.79 0.33]; % SI units, all 4 layers % <- from Oostendorp
+                    headmodel.conductivity = [0.33 0.0041 1.79 0.33]; % SI units, all 4 layers
+                    headmodel.conductivity = [0.33 0.022 1.79 0.33]; % SI units, all 4 layers % <- from Oostendorp
                 case 3
-                    headmodel.cond = [0.33 0.0041 0.33]; % SI units, ignore CSF
-                    %                headmodel.cond = [0.33 0.022 0.33]; % SI units, ignore CSF % <- from Oostendorp
+                    headmodel.conductivity = [0.33 0.0041 0.33]; % SI units, ignore CSF

```

Comment:
I feel all those ``headmodel`` should be ``cfg`` as well as only ``cfg`` and ``headmodel.bnd`` get passed below and the ``headmodel`` gets overwritten by ``ft_prepare_headmodel`` in line 65 ?

Reply:
This seems to be fine on second look, as we construct the headmodel without help from Fieldtrip. I clarified that in a comment.